src/treetensornetwork/opsum_to_ttno.jl

```diff
@@ -0,0 +1,594 @@
+# convert ITensors.OpSum to TreeTensorNetworkOperator
+
+# TODO: fix symbolic SVD compression for certain combinations of long-range interactions!
+
+# 
+# Utility methods
+# 
+
+# number of neighbors of vertex in graph
+function num_neighbors(graph::AbstractGraph, vertex...)
```

Comment:
Looks like this is the same as `degree`: https://juliagraphs.org/Graphs.jl/dev/core_functions/core/#Graphs.degree. Hopefully `Graphs.degree` already works for `AbstractNamedGraph` and `AbstractDataGraph`.

Reply:
Good catch, completely missed this. `Graphs.degree` works fine now thanks to mtfishman/NamedGraphs.jl#17 and mtfishman/DataGraphs.jl#7.